### PR TITLE
Add CD configuration and documentation

### PR DESCRIPTION
**Pull request by Gabrielpanga**
Add CD configuration and documentation
Issue: #40

### CODE REVIEW

File: docker-compose.prod.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+version: '3'
+
+services:
+  app:
+    image: gabrielcodelitt.azurecr.io/nodestartercodelitt_app:latest
+    ports:
+      - '80:8080'
+    restart: always
+    environment:
+      SERVER_PORT: 8080
```
Comment (kaiomagalhaes) [inline docker-compose.prod.yml]: Any reason to not use environment variables here?

Comment (Gabrielpanga) [reply docker-compose.prod.yml]: You mean azure application settings? This is production ready docker compose settings.

Comment (kaiomagalhaes) [reply docker-compose.prod.yml]: yeah but we shouldn't hard code the environment, but pass this on the deploy

Comment (Gabrielpanga) [reply docker-compose.prod.yml]: Updated.

**Pull request merged**